notebook/analysis/benchmark_results.ipynb

```diff
@@ -1497,6 +1501,604 @@
y+QZMtSCiuHhYfT09FRd100egigi+MXPgg8N/q7+XZAONV7kZqJhX18fFi9eXGu1IpzR7zAa/Ap5i4dt6bqOoYEh9PbOcSwXeoW61oYmSNmwNKpj8FqNd2x0HLHWporlQquI4S3Vn4USwS/Xvge3XHWQSw+WvRs2bICYECHPc5eICK0iErOS2LBhQ3WCTyk026CKRUHwbUn/5rJqGBgegHhQjQh+V0k6OGPGDI+tCB7Nr3gn1Ijgj4yMQG4vOUjVgtAhYnSkkuCbgxa9jOAzvYB4rL70xw2VTkDVXXRGR0cht1fe/24Q20Xseck5cO/p6cH++y/H+PCjFQR/PPUo5sxZwAtccXCExF6X6ORyOTz88MN4+9vfjmQyif322w9r1qzBfffdV3fde++9FyeccIIjYt8IUIEGiuDrugbiNqsp1Cd2xhRwZcQsVJKtfV2PBGnPyB7LQcdqW7s3iY6qKo4kW0IkaFo4H3xKiCfNrgkz2UueWYpyid3efLNLII6/gpJLo7gPsa6DCG8RXFVVsad/D6RZxjEIzQLk1hheffXVAK0IL1EgZf/Ww+DgIFRFrbDYo7NFbN25rea6bhKdoM+Atc2yQUq966lpGsZHxiF0VhJMsVPC4J7q+RDWILGC4BvbCjrgNRV2XhL+N2zYgMSSppoEU1oSwwsvbqj6vZGHZBuMFPsm05/dL3TmlHoBtQc7I8MjEDuqR8DN3KB6kq+oYBD80mcdxVkOF4yPj0Ns8TarILSKUPJKRZ0UQggkSYauV0bwg74PqYsG3+2dxhjD5PikVeywHoRWEeMu1c5PPvl4ZPKPVyzP5p/A6tUneGw1BwdHNex1gr9rl6EvnD9/vrVs8eLF2L59e831Jicn8dhjj+HUU0+tWP7Od74Tl112Gb73ve/VLToTBJQKFZ25sbz26dRUDURweakK9cmpEaWLLmLmiOD7sMkcHhmuIPhopxiu42HOGIOmqWUafAmapgYmyGbUz0vEz0R/f3+F+4bpm213eqi2v+IapYUkeHEcVVUh2K6f1+hxf38/dE13uHDE5sSwY8eOuuuGiXS7wYh+l1x0vBRM27p1K8SkBLHbSXJi8+LY5mIJWB/hCL7OWCl6jPr30e7du8F03b3I0kwZwwNDVYl6advlzy0t+9473PqAWgT/+Zeeh7h/bRlHbGkCL9RItBUEwSL1gFFgyVoeALrNyci8FrXuVXuxPTcQ0cjpqOfKFBUqI/jVJTrpdBrUpUiaG4QktdYphyzHwZgzoVtneSQSQSP45XkE7hH8fD4PtaCCtngTAAgtAlITlS5rxx57LNKZXcgXSv2uoo5iMr0Zq1at8n8AHBwcDux1iU4ul0Mi4Uz2ampqqlssZt26dZg5cyaWL19uLZs7dy6+/e1vY+7cuRgZGcH3v/993HDDDbjmmmsc6/b39ztsEWOxmK9sfVESodliHVqxV5RlueYLTtf0kp7EBiIYxK7WuqIoOqPsxZdrvX1WA7ERYyNFj3nazvjoOJrbO5xtaxcxvHG45vpu2mPzb13XAxczsUfwCSF1j2FgYADxGZUFZqQZhpVbrfWtF7ZDw+5tv25gjFn6e6BY4KfOfQAAO3bsMDyobQSHzhOxdfvWuutSSp3Hbjr4BCRmxibsCar1t7Vp0yYkFlZeg9iiBAbGXsXo6Ci6u7td16217aDHYB/s0uKMSq1tbd++HVKT5BrBj82PQ1M19PX1YdGiRRXfl/JenMdOfNzD5RAEwSKXJuUWRdF1O5qmYfPGzeg+s3YRq/h+Cezs70M6nUZra2vF96IoOiL4ZsAhaH9kT3QuBR7cz0Uul0MhV6hJ8AFAapMwMTFRtT2VPbFByoO0X5Ik6z0AABqqX4NCoeBqseraxmIit6IoFduS5ZiLRCePRCIe6BioIFgzQIARwXc7BnM2gSa9DVJoUkA2k63YzoIFC9DTMwupzHrE5DMAAKnMM2ht7cCyZctCz8xxcLzesdcJfjweryDz6XS6gvSX47777quI3nd0dKCjwyCf3d3duPzyy3HllVcin887pi1vvPFGXH/99dbnz3zmM/jiF7/oo80J1wh+Z2cnmpubXdcBitFOtz6RGi8Is+1uaG1tdY3gd3Z2oq2tzXPbTTQ1NTleqAyouX/AIOnpiTTa2p06daFdxNj4WM31M5mMsS8Xgt/U1FTzvFVDMpk0CH7xPdDU1FT3GEZGRkB7XV5+vRR7hvfUXN+VnBHiab9uSCQSjgi+QAhkWa67rd27dyM2z0mQ5bkxbL1va911k8kkyqkNISRQ+wGD0Nm3xhiru60NL2+A4BJBlmbJkJpkbN++HUuWLHFdt5rLDCU08DFQSh0EPx6P19zWli1bEN8v6UpAhFYRiZ4kduzYgSOOOKLi+4mJYgJuRUdgbKu1tdX3cbS1tVnRbpNjtrW1uW5ny5YtUPIKYvvV7l/leXFQSUB/f79rhedYTHZKBot2jV1dXXUTvd1gLzll/lvtXJjBmXoSEbFVRD6fr3o+qSBAt0+0hHiW29raHBF8jTHEYu73kSAIJW/cOjBd15LJZMW2kskk9IJTuqOzPNrb3a99PcRisQoXpubm5optWVW03VykXEBiFGpBRWtrawXJP/HE4/D3+14AYBD8TO4FHHvsKnR2drpsiYODww/2OsGfM8dItNu5c6eVVLN161bXl4qJLVu2YMeOHTjllFNqbtssDlU+1XvFFVfgnHPOsT7HYrG6/tt2qJoGoWw6FjBe3rVs7qpF8CEYg5pabchms2B6peZ1cnIy0LT+5OSkM4Kvs7rnYHh42Ig6l0XOhDYRo8NDNdc3iY2z0JWxnaGhoUD2gJOTk6CkJHYYGxurewybt20G7akkBlKvhC1Pbqm5fmm635mkmkqlfN0/JsbHx2GXQVMwT9t65vlnK6pgygsS6Ht1K/r7+2sSrEwmU0bMDFIepP2AEc0r106PjIxUjb6pqornn3se3adURpAJJYgvS+CRRx5xJM7b4V6xl0FneuBj
```

Comment:
Right, pytorch supervised is the shallow network. It's named that way because it was the first model using PytorchSupervised, but I need to clarify that at some point